packages/gestalt/src/Typography.css

```diff
@@ -26,10 +26,6 @@
 
 /* leading */
 
-.leadingShort {
-  line-height: 1.2;
-}
-
 .leadingTall {
```

Comment:
@AlbertCarreras should this also get removed?

Reply:
DONE